components/skills/SkillsHeading.tsx

```diff
@@ -1,11 +1,22 @@
+'use client';
+
+import { fadeIn } from '@/utils/motion';
+import { motion } from 'framer-motion';
+
 const SkillsHeading = () => {
   return (
-    <h2 className="mb-[2.5rem] text-center text-[2.25rem] font-bold leading-[2.6rem] tracking-[-0.36px] text-black-200 dark:bg-black-200 dark:text-white-900 md:text-[3rem] md:leading-[3.45rem] md:tracking-[-0.48px]">
+    <motion.h2
+      variants={fadeIn('right', 'spring', 0.5 * 1, 2.75)}
+      viewport={{ once: true, amount: 0.1 }}
+      initial="hidden"
+      whileInView="show"
+      className="mb-[2.5rem] text-center text-[2.25rem] font-bold leading-[2.6rem] tracking-[-0.36px] text-black-200 dark:bg-black-200 dark:text-white-900 md:text-[3rem] md:leading-[3.45rem] md:tracking-[-0.48px]"
+    >
```

Comment:
Can we not make this a reusable component? I see you're using it below inside components/testimonials/TestimonialsHeading.tsx